content/Hardware Support/MKR Family/About-the-IoT-Bundle-projects.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "About the IoT Bundle projects"
+id: 4407181791762
+---
+
+Both **Arduino IoT Bundle** and **Arduino _MKR_ IoT Bundle** include components that can be used for five projects.
+
+---
+
+## Arduino IoT Bundle
+
+Tutorials for the **Arduino IoT Bundle** projects are available in Arduino Docs:
+
+* [I Love You Pillow](https://create.arduino.cc/projecthub/Arduino_Genuino/i-love-you-pillow-with-the-arduino-iot-bundle-cec4c4)
+* [Puzzle Box](https://create.arduino.cc/projecthub/Arduino_Genuino/puzzlebox-with-arduino-iot-bundle-96d09c)
+* [Pavlov's cat](https://create.arduino.cc/projecthub/Arduino_Genuino/pavlov-s-cat-with-arduino-iot-bundle-d5b388)
+* [The Nerd](https://create.arduino.cc/projecthub/Arduino_Genuino/the-nerd-with-arduino-iot-bundle-b1d0ca)
+* [Plant Communicator](https://create.arduino.cc/projecthub/Arduino_Genuino/plant-communicator-with-the-arduino-iot-bundle-918636)
+
+These projects use Arduino IoT Cloud and the Arduino IoT Cloud Remote app (and can be built using the free Arduino Cloud plan).
+
+## Arduino MKR IoT Bundle
+
+Instructions for the **Arduino _MKR_ IoT Bundle** projects are available in Arduino Project Hub:
```

Comment:
Maybe the link to project Hub as a hyperlink, but since the Project hub is being migrated are these projects also being migrated.

Reply:
The links are for the migrated projects, so we should be good.